website_portal_v10/README.rst

```diff
@@ -82,6 +82,10 @@ welcomed feedback.
 Credits
 =======
 
+**This module is a backport from Odoo SA and as such, it is not included in the
+OCA CLA. That means we do not have a copy of the copyright on it like all
+other OCA modules.**
+
```

Comment:
Dear @OCA/board is this OK? Could we merge? I'm not sure in this point. https://github.com/OCA/website/pull/204#issuecomment-232560496